Assignment9.rb

```diff
@@ -0,0 +1,20 @@
+# Write a class (with constructor) to define instance methods for getting area & perimeter of a rectangle.
+class Rectangle
+  def initialize(length, width)
+    @length = length
+    @width = width
+  end
+  
+  def area
+    @length * @width
```

Comment:
can you use `attr_accessor`

Reply:
instead of initialize?

Reply:
> can you use `attr_accessor`
instead of initialize?

Reply:
@Subhajit-QI please read about getters and setters methods and how to use that https://www.geeksforgeeks.org/ruby-getters-and-setters-method/